CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - ([#331](https://github.com/ramsayleung/rspotify/pull/331)) `Market` is now `Copy`
 - ([#366](https://github.com/ramsayleung/rspotify/pull/366)) Replace all `std::time::Duration` with `chrono::Duration` to support negative duration.
+- ([#375](https://github.com/ramsayleung/rspotify/pull/375)) We now use
```

Comment:
Since `v0.11.6` has released, these changes should be record for the `v.0.11.7`

Reply:
Fixed. Shall I merge the PR, then?

Reply:
Sure, I have merged this PR